hs-bindgen/fixtures/globals.pp.hs

```diff
@@ -71,37 +71,37 @@ instance F.Storable Inline_struct where
                F.pokeByteOff ptr0 (0 :: Int) inline_struct_x2
             >> F.pokeByteOff ptr0 (4 :: Int) inline_struct_y3
 
-foreign import capi safe "&compoundGlobal2" compoundGlobal2 :: F.Ptr Inline_struct
+foreign import ccall safe "get_compoundGlobal2_ptr" compoundGlobal2 :: F.Ptr Inline_struct
 
-foreign import capi safe "&nesInteger" nesInteger :: F.Ptr FC.CInt
+foreign import ccall safe "get_nesInteger_ptr" nesInteger :: F.Ptr FC.CInt
 
-foreign import capi safe "&nesFloating" nesFloating :: F.Ptr FC.CFloat
+foreign import ccall safe "get_nesFloating_ptr" nesFloating :: F.Ptr FC.CFloat
 
-foreign import capi safe "&nesString1" nesString1 :: F.Ptr (F.Ptr FC.CChar)
+foreign import ccall safe "get_nesString1_ptr" nesString1 :: F.Ptr (F.Ptr FC.CChar)
 
 foreign import capi safe "&nesString2" nesString2 :: F.Ptr ((HsBindgen.Runtime.ConstantArray.ConstantArray 3) FC.CChar)
```

Comment:
Note to self. We're _not_ generating a wrapper for constant arrays. We could probably avoid the wrapper in more cases (probably for all global variables that are pointers); however, the only place where it's really _useful_ to not generate the wrapper is constant arrays, because it's harder to keep type information in that case. So for now we'll just treat this as the exception to the rule; we might reconsider this later.